net-dns/dnsdist/files/dnsdist.conf

```diff
@@ -0,0 +1,2 @@
+-- This is a dnsdist configuration file.
+-- Please refer to http://dnsdist.org/README/ for more information.
```

Comment:
I'll put some real example there, and rename the file to `dnsdist.conf.example`.

I'll shall mention, that if you came to use dnsdist, you are kind of hardcore DNSed enough to understand what a tool that is and which power it brings with. That's not for everyone, albeit it is useful in default setups (whatever that might be), too.